src/Tags.zig

```diff
@@ -1,5 +1,9 @@
 const std = @import("std");
 
+const windowsApi = @import("windows.zig");
```

Comment:
Name is horrible and more broadly no idea if you want to do a whole module for those declarations, I just did it cuz it seemed cleaner at the time or something

Reply:
Why not just `windows`?